packages/dd-trace/src/config.js

```diff
@@ -917,11 +910,68 @@ ken|consumer_?(?:id|key|secret)|sign(?:ed|ature)?|auth(?:entication|orization)?)
     }
 
     this.sampler.sampleRate = this.sampleRate
-
     updateConfig(changes, this)
+
+    return changes
+  }
+
+  _getConfigValue (name) {
```

Comment:
Should the config options be stored directly with the `.` as a flat key-value pair instead of using nested objects if that's the expected format? Not sure how this would impact things internally though, so might be safer to keep this helper, but just a thought.

Reply:
In any case, let's move these helpers elsewhere as they're general helpers.

Reply:
we could start storing them as flat key-value pairs in the future. Right now [some values](https://github.com/DataDog/dd-trace-js/blob/5fe741040e4cb7b4d0adc6f13e12cfdad246fbf1/packages/dd-trace/src/config.js#L355-L371) are still set in the old way